### PR TITLE
Add multiple node support for Jenkins slaves.

### PR DESCRIPTION
**Pull request by s1061123**
To support multiple node, the following changes is done in commit.
- move jenkins parameter from ~/.ansible/var/cira_vars.yml to
  inventry variables.
- use loop (i.e. with_items) to add nodes in jenkins
- modify templates to support multiple nodes

### CODE REVIEW

**Comment by s1061123** [comment]
I've tested with following environment and verified two slave is up from scratch.

===== hosts/jenkins_slave
[jenkins_slave]
nfvsdn-03 ansible_host=10.8.125.13 ansible_user=ansible
tohayash-slave1 ansible_host=10.3.9.128 ansible_user=ansible

[jenkins_slave:vars]
slave_description=CIRA Testing Node
slave_remoteFS=/home/jenkins
slave_port=22
slave_credentialsId=jenkins-credential
slave_label=cira

File: README.md [hunk 1]
```diff
@@ -185,6 +177,13 @@ with the following template:
 
     [jenkins_slave]
     slave01 ansible_host=10.10.1.1 ansible_user=ansible
+ 
+    [jenkins_slave:vars]
```
Comment (leifmadsen) [inline README.md:181]: I think I like this location better, yes :)

**Review by leifmadsen — COMMENTED** [review]
So this is a _really_ good first start! I need to check, but I'm pretty sure the SSH stuff isn't going to work as it should.

In the `jenkins_ssh.yml` file there is some fingerprinting and SSH key distribution stuff that also needs the same type of work done to it.

To me so far though, looks correct. Just a bit more to do. I might take a stab at it today using the same `with_items` and group methods you'd employed here.

Great work!

**Comment by s1061123** [comment]
Leif,

In jenkins_ssh.yml, jenkins_slave is called twice (or more. depends how many node weitten under jenkins_slave) for each host in the group jenkins_master of inventory, hence nothing is required for ssh key distribution for master as far as I understand.

If we need to distribute from one slave to another, we need to add more code.

**Comment by leifmadsen** [comment]
@s1061123 huh... maybe you're right :)

Actually, yes, now that I think about that more, you're totally right. It's going to run the same stuff for each node, so it actually already supports this.

Between slaves, correct, we don't need that. We're basically doing a hub and spoke topology here, so only the master needs to communicate with all the slaves, but we don't need to be full mesh.

I've merged this down, and I'll play today. Great work! Thanks for adding!

**Pull request merged**